browser/ui/BUILD.gn

```diff
@@ -173,6 +177,7 @@ source_set("ui") {
     "//chrome/common",
     "//components/gcm_driver:gcm_buildflags",
     "//components/prefs",
+    "//components/sessions",
```

Comment:
looks like we also need `//ui/views/` or is that already here somewhere?

Reply:
Added.